ocs_ci/ocs/fiojob.py

```diff
@@ -143,8 +143,21 @@ def wait_for_job_completion(namespace, timeout, error_msg):
     except TimeoutExpiredError as ex:
         # report some high level error as well
         logger.error(error_msg)
-        # TODO: log both describe and the output from the fio pods, as DEBUG
         ex.message = error_msg
+        # fetch log(s) of any fio pod(s) in the job namespace
+        pod_data = ocp_pod.get()
+        for pod_dict in pod_data.get('items', []):
+            pod_name = pod_dict['metadata']['name']
+            output = ocp_pod.get_logs(pod_name)
+            if len(output) == 0:
+                logger.error(
+                    "Container log from pod '%s' is empty.", pod_name)
+            else:
+                logger.error(
+                    "Container log from pod '%s' follows:\n%s",
+                    pod_name,
+                    output)
```

Comment:
@mbukatov please fix all the closing parenthesis in this PR (that you changed)

Reply:
Tracking via https://github.com/red-hat-storage/ocs-ci/issues/1892